<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ftoc">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>200</y>
      <width>125</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>F -&gt; C</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="ctof">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>250</y>
      <width>125</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>C -&gt; F</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tempInputBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tempOutputBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="convertSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="value">
     <number>49</number>
    </property>
    <property name="sliderPosition">
     <number>49</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="introLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to D&amp;D's Lab18 converter!
This program converts temperature from
 fahrenheit to celsius and vice versa!
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
